pages/about/product_updates.md

```diff
@@ -13,21 +13,45 @@ eleventyNavigation:
+
+Check out the site and [let us know what you think](https://forms.office.com/g/Uq30UkMYRu). Here are a few places to start.
+
+- [Why you should use .gov](../../domains/benefits/)
+- [How to move to .gov from another domain](../../domains/moving/)
+- [Steps to take before you request a .gov domain](../../domains/before/)
+
+
 ## November 17, 2023
 
-Our new domain management system—the .gov registrar—is now available. The organization, contacts, and DNS information for all .gov domains have been transferred to the new registrar. 
+Our new domain management system—the .gov registrar—is now available. We’ve transferred organization, contacts, and domain name system (DNS) information for all .gov domains to the new registrar.
 
-If you have questions or concerns about the information you see for your domain, email us at <help@get.gov>.
+[Let us know](https://forms.office.com/g/Uq30UkMYRu) if you have questions or concerns about the information you see for your domain.
 
 ### Changes from the previous registrar
-- We no longer have the admin, technical, and billing roles. In the new registrar, these contacts are all considered “domain managers,” and you aren’t limited to three domain managers like in the old system.
-- We’re now using Login.gov to manage access to the registrar. Login.gov is a simple and secure process for signing in to many government services with one account.
+
+- We no longer have the admin, technical, and billing roles. In the new registrar, these roles are all considered "domain managers." You can add an unlimited number of domain managers. 
+- We’re now using [Login.gov]((https://login.gov/){.usa-link--external} to manage access to the registrar. Login.gov is a simple and secure process for signing in to many government services with one account.
 
 ### Some domain information won’t immediately be available in the registrar
-- Domain expiration date: Good news! Expiration dates for .gov domains in good standing have been extended for one year.
-- Organization address: You’ll see a place for your organization’s mailing address in the new registrar, but we haven’t transferred this data into the system yet.
+
+- Domain expiration date: Good news! We have extended expiration dates for .gov domains in good standing for one year.
+- Organization address: We have not yet transferred your organization's mailing address to the new registrar. There is a place to save that information in the new registrar, but it is empty for now.
 
 ### Domain management features that aren’t enabled yet
-- Email notifications won’t be sent for domain information changes (like a change to your contact information).
-- A change history for domain information won’t be available.
+
+- Email notifications aren’t sent for domain information changes (like a change to your contact information).
+- A change history for domain information isn’t available.
 - Domain managers can’t remove another user’s access without contacting us.
```

Comment:
Once Zander's PR is merged and we do another release, domain managers will be able to remove another user's access...Might we want to communicate that this feature will be available...soon?

Reply:
Hmm, okay. Will do.